src/Regridder.jl

```diff
@@ -64,6 +65,33 @@ function reshape_cgll_sparse_to_field!(field::Fields.Field, in_array::Array, R)
     Spaces.dss!(Fields.field_values(field), topology, hspace.quadrature_style)
 end
 
+function reshape_cgll_sparse_to_field!(
+    field::Fields.Field,
+    in_array::SubArray,
+    R,
+    space::Spaces.ExtrudedFiniteDifferenceSpace,
+)
+    field_array = parent(field)
+
+    fill!(field_array, zero(eltype(field_array)))
+    Nf = size(field_array, 4)
+    Nz = size(field_array, 1)
+
+    for z in 1:Nz
```

Comment:
and also maybe a comment here saying that this is the main difference between the 2d and 3d cases